BowlingBall.Tests/GameFixture.cs

```diff
@@ -1,14 +1,83 @@
 using System;
 using Xunit;
+using BowlingBall;
+//using Moq;
 
 namespace BowlingBall.Tests
 {
     public class GameFixture
     {
+
         [Fact]
-        public void DummyTest()
+        public void Game_when_UserRollsValidInput()

```

Comment:
expectation missing in name
the format that I mentioned was:
`Game_when_{condition}_should_{expectation}`